[FIX] ESBEP-26631 bank payment move memory performance

PieterPaulussen opened the pull request.

StefanRijnhart commented on account_banking_sepa_direct_debit/wizard/export_sdd.py:
```diff
@@ -372,7 +395,8 @@ def save_sepa(self):
             first_mandates = abmo.browse([])
             all_mandates = abmo.browse([])
-            for bline in order.bank_line_ids:
+            bl_ids = order.bank_line_ids.ids
```
order.with_context(prefetch=False).bank_line_ids.ids

StefanRijnhart commented on account_banking_sepa_direct_debit/wizard/export_sdd.py:
```diff
@@ -353,6 +353,29 @@ def create_sepa(self):
             xml_root, total_amount, transactions_count_1_6, gen_args)
 
+    @api.model
+    def chunked(self, records_or_ids, model=None, size=PREFETCH_MAX,
```
Ik had deze method al op parent class 'banking.export.pain' gezet.

Commit:
[FIX] ESBEP-26631 improve payment order validation

StefanRijnhart committed:
[WIP] ESBEP-26631, circumvent the ORM while reconciling payment lines
in a desparate attempt to increase the performance (with a result of ca. 90%
performance increate in this method). Some breakage expected.

PieterPaulussen commented on account_banking_pain_base/models/banking_export_pain.py:
```diff
@@ -39,10 +39,14 @@ def chunked(self, records_or_ids, model=None, size=PREFETCH_MAX,
         length = len(ids)
+        if note is None:
+            note = ''
+        if note:
```
`if note and note != '':`

PieterPaulussen replied:
NEVERMIND!

PieterPaulussen commented on account_banking_payment_transfer/model/account_payment.py:
```diff
@@ -223,10 +228,14 @@ def chunked(self, records_or_ids, model=None, size=PREFETCH_MAX,
+        if note is None:
+            note = ''
+        if note:
+            note = '%s: ' % note
```
Same as above.

PieterPaulussen reviewed (commented):
Minor suggestions. Otherwise this looks very good.

PieterPaulussen reviewed (commented):
Can't approve my own PR. @StefanRijnhart go ahead!
